third-party-audit/sig-security-external-audit/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
@chetak123 After renaming `third party audit` to `sig-security-external-audit`, can you please move these 3 files (owners, readme, and roadmap) up one level and remove the resulting empty directory?